gcp_variant_transforms/transforms/vcf_field_conflict_resolver.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import vcf
+
+__all__ = ['FieldConflictResolver']
+
+class FieldConflictResolver(object):
+  """A class for resolving all VCF field related mistmatches."""
```

Comment:
Is this functionality only going to be used for header merging? If yes, dose it help to reference "header" somewhere in the comment or even more preferably in the name of the class? Or maybe even the name of the module, should have "header" in it?

Reply:
It's not specific to header, and is going to be used for resolving conflicts between schema and field_data as well (in my next PR).

Reply:
Hmm, okay then I think this warrants adding more documentation either at module level or here, maybe with some examples of how you expect this to be used in the Header and non-Header cases. It was not obvious to me that this is not header specific because for example how you compare keys, types, etc. which seemed header specific.

I am sure in your next PRs this become more clear, so feel free to add the documentation there (but please add a TOOD now not to forget).